packages/lab/src/Flow/Node/Parameters/Slider.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react";
+
+import { css } from "@emotion/css";
+
+import { useReactFlow } from "reactflow";
+
+import { HvSlider, HvSliderProps } from "@hitachivantara/uikit-react-core";
+
+import { HvFlowNodeSliderParam } from "../../types";
+
+interface SliderProps {
+  nodeId: string;
+  param: Omit<HvFlowNodeSliderParam, "type">;
+  data: any;
+}
+
+const classes = {
+  labelContainer: css({
+    marginRight: 0,
+    marginLeft: 0,
+  }),
+  sliderBase: css({
+    padding: 0,
+  }),
+};
+
+const Slider = ({ nodeId, param, data }: SliderProps) => {
+  const { id } = param;
+
+  const reactFlowInstance = useReactFlow();
+
+  const [value, setValue] = useState(data[id]);
+
+  const onSliderchange: HvSliderProps["onChange"] = (val) => {
```

Comment:
We shouldn't need to control the value of the element internally, we could use `defaultValues={data[id]}`

Also, typo on the capitalisation of `onSliderChange`

```suggestion
  const onSliderChange: HvSliderProps["onChange"] = (val) => {
```